Instructions/Labs/LAB[MB-240]_Lab03_Resource_Configuration.md

```diff
@@ -70,7 +70,7 @@ In this exercise, you will create the location where resources will start and en
 
 1. Select **User (1)** from the **Resource Type** drop-down field.
 
-1. Select the user record you are signed in as ex. **ODL User [DeploymentID]** for the **User (2)**.
+1. Select the user record you are signed in as ex. **ODL_User [DeploymentID]** for the **User (2)**.

```

Comment:
@aparnakhadatkar @junnhssn , we were planning to not include Deployment ID can you check

Reply:
Sure @akn9050

Reply:
In Some cases, we have to select the username  @akn9050   so we updated it like this.

Reply:
Amit, We are using this when we are asking the user to select the ODL user as part of the lab.